delay-io.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * @function steal-socket.delay-io delay-io
```

Comment:
I was thinking about this as a public. You can import only this function if you want just to delay socket.io on client.